adagrams/game.py

```diff
@@ -1,11 +1,136 @@
+import random
+import copy
+
 def draw_letters():
-    pass
+    
+    letter_frequency = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    
+    letter_pool = []
+    letter_bank = []
+    #generate elements for letter_pool list in correct frequency
+    for letter, frequency in letter_frequency.items(): 
+        for _ in range(frequency):
+            letter_pool.append(letter)
```

Comment:
👍 Nice way to build up the list of all "tiles". I especially like the use of `_` as the throwaway variable name.

Rather than doing the inner loop ourself, we could instead extend the letter pool with a string made by repeating the current letter like this:
```python
    for letter, frequency in letter_frequency.items(): 
        letter_pool += letter * frequency
```

The result would be the same.